content/en/logs/guide/control-sensitive-logs-data.md

```diff
@@ -16,100 +16,94 @@ further_reading:
+Once sensitive data in logs is sent to the Datadog platform, it may exist in a number of places. As a result, check each of the following (ordered from most likely to have sensitive data to least likely):
 
-* Check your Log [Archives][6], which is where Datadog sends logs to be stored on your end. Set up Archive Filters to see if your archive contains sensitive logs.
+* Datadog [Indexes][3] are where logs are stored in Datadog until they age out according to index retention. Focus should be on Datadog Indexes as other locations are less likely to be a compliance concern. Check [indexes filters][4] and [exclusion filters][5] to see if logs with sensitive data are indexed.
 
-* Datadog [Metrics generated from Logs][7] storing aggregated metrics, meaning sensitive data likely have been discarded or crushed. Check custom metrics filters to see if logs with sensitive data are processed.
+* Log [Archives][6], which is where Datadog sends logs to be stored. Set up Archive Filters to see if your archive contains sensitive logs.
 
-* Datadog [Log Monitors][8] notifications when they include [Log Samples][9]. Check specifically for monitors triggered during the period when sensitive data was being sent. 
+* [Metrics generated from logs][7], which stores aggregated metrics. Sensitive data may have been discarded with this process. Check custom metrics filters to see if logs with sensitive data are processed.
 
-* Datadog [Livetail][10], where logs are streamed in web browsers from your organization's users. There is no persistence beyond the 50 logs cached in browsers, and for broader queries, the result can be highly sampled.
+* [Log Monitors][8] notifications when they include [Log Samples][9]. Check specifically for monitors triggered during the period when sensitive data was being sent.
 
+* [Livetail][10], where logs are viewed in real-time by your organization's users. There is no persistence beyond the 50 logs cached in browsers, and for broader queries, the result can be highly sampled.
 
 ## Fix the source of the data upstream
 
 ### Stop indexing sensitive logs
 
 This is an optional step to prevent additional sensitive data from being sent to Datadog. However keep in mind that you’ll still need to address the sensitive data already sent to Datadog.
 
-* Find which index(es) logs holding sensitive data are susceptible to be routed.
-* For each index, add an exclusion filter based on the sensitive outline query. It has to be **in the first place**. 
-
-{{< img src="logs/guide/sensitive/sensitive_exclusion-filters.png" alt="Sensitive Exclusion Filters" style="width:80%;" >}}
+* Find which index(es) that hold sensitive data are susceptible to routing.
+* For each index, add an exclusion filter based on the sensitive outline query.
 
+{{< img src="logs/guide/sensitive/sensitive_exclusion-filters.png" alt="Sensitive Exclusion Filters" style="width:80%;" >}
```

Comment:
I think you might be missing a slash or something in this tag and that's what's making it fail?

Reply:
Got it, was a missing `}`